.travis.yml

```diff
@@ -19,4 +19,4 @@ before_install:
   - bash setup-travis.sh
 script:
   - sbt clean test package buildAll
-  - LD_LIBRARY_PATH=libhttpsimple/target cli/target/scala-2.11/reactive-cli-out --help
+  - LD_LIBRARY_PATH=cli/target/scala-2.11/reactive-cli-out --help
```

Comment:
LD_LIBRARY_PATH= should be removed, i.e. line should be just `cli/target/scala-2.11/reactive-cli-out --help`